javascript/slides/varsfunctions.html

```diff
@@ -196,7 +196,34 @@
                                                             </section>
                                                         </div>
 
-
+                                                        <div class="slide normal"> 
+                                                        <header><h1>Strings</h1></header> 
+                                                         <section class="content"> 
+                                                           <p>A string holds an ordered list of character:
```

Comment:
oh RIGHT I forgot that we introduce Arrays later.... that makes sense then. Maybe we can add an additional note about what an `index` is and where students can go to find more string methods:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/prototype

We can also verbally remind them of the similarity when we get to Arrays later.